interpreter/chapter.txt

```diff
@@ -131,9 +137,10 @@ Remember that given valid instruction sets, without any changes to our interpret
         "numbers": [7, 5, 8] }
 ~~~~
 
-At this point, we can begin to see how this structure is extensible - we can add methods on the interpreter object that describe many more operations (as long as we have a compiler to hand us well-formed instruction sets).
+At this point, we can begin to see how this structure is extensible: we can add methods on the interpreter object that describe many more operations (as long as we have a compiler to hand us well-formed instruction sets).
+
+#### Variables
 
-### Variables
 Next let's add variables to our interpreter. Variables require four pieces: an instruction for storing the value of a variable, `STORE_NAME`; an instruction for retrieving it, `LOAD_NAME`; and a mapping from variable names to values. For now, we'll ignore namespaces and scoping, so we can store the variable mapping on the interpreter object itself. Finally, we'll have to make sure that `what_to_execute` has a list of the variable names, in addition to its list of constants.
```

Comment:
For the record, I think the fourth part was the list of variable names that `what_to_execute` requires, which is the last sentence in the paragraph.